website/static/css/pages/login-page.css

```diff
@@ -38,3 +38,12 @@
     background-color: #CCC;
     border-color: #BBB;
 }
+.existing-user-signin {
+    /*height: 320px;*/
+    background-color: #EEE;
+    border-radius: 4px;
+    margin-top: 75px;
```

Comment:
Why all this extra margin? We should rarely need to override Bootstrap's defaults. In this case, we can remove both the `border-radius` and `margin-top`.

Before:

![osf___sign_in_and_login-page_css____projects_cos-projects_osf_website_static_css_pages__-_vim1](https://cloud.githubusercontent.com/assets/2379650/15757807/68cd6f56-28d5-11e6-874a-cdcdde1d1146.png)

After:

![osf___sign_in_and_login-page_css____projects_cos-projects_osf_website_static_css_pages__-_vim1](https://cloud.githubusercontent.com/assets/2379650/15757827/7b7ca2c0-28d5-11e6-80e4-36e17b96f336.png)